ricecooker/commands.py

```diff
@@ -292,10 +293,12 @@ def create_tree(tree):
     """
     # Create tree
     config.LOGGER.info("\nCreating tree on Kolibri Studio...")
+    # channel_id, channel_link = tree.upload_channel_structure()
```

Comment:
Is this meant to be commented out? i.e. we're not using `upload_channel_structure` now, but will be used in the future?